modules/config/main.tf

```diff
@@ -61,6 +61,24 @@ resource "aws_config_delivery_channel" "mgmt_config_delivery_channel" {
   depends_on = [aws_config_configuration_recorder.mgmt_config_recorder]
 }
 
+resource "aws_config_configuration_recorder_status" "mgmt_config_recorder_status" {
+  name       = aws_config_configuration_recorder.mgmt_config_recorder.name
+  is_enabled = true
+  depends_on = [aws_config_delivery_channel.mgmt_config_delivery_channel]
+}
+
 resource "aws_config_retention_configuration" "mgmt_config_retention" {
   retention_period_in_days = var.config_retention_in_days
 }
+
+resource "aws_cloudformation_stack" "mgmt_config_rules_cloudformation_stack" {
+  for_each = var.config.rule_groups
+
+  name   = format("%s%s", var.config.stackset_name_prefix, lower(each.key))
```

Comment:
Are you sure the stack name is unique? Are you sure on expectation on the stack name?

Reply:
Have you checked AWS console - a precheck - to confirm your expectation on the stack name.